lib/lib.c

```diff
@@ -45,6 +45,388 @@ base64_table_dec[] =
 	255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
 };
 
+#if BASE64_LITTLE_ENDIAN
+
+/* SPECIAL DECODE TABLES FOR LITTLE ENDIAN (INTEL) CPUS */
```

Comment:
Code from Nick Galbreath. You can find the generator for the table at client9/stringencoders